apps/server/src/shared/domain/entity/user.entity.ts

```diff
@@ -54,7 +54,7 @@ export class User extends BaseEntityWithTimestamps implements IEntityWithSchool
 	@Index()
 	ldapDn?: string;
 
-	@Property({ nullable: true })
+	@Property({ nullable: true, fieldName: 'ldapId' })
```

Comment:
Why do you create it?

Reply:
we plan in accordance with ErWIn to rename this field (at least internally) to "externalId". for compatibility with feathers we keep it as ldapId in the database.